packages/blocks/src/__snapshots__/elements.spec.ts.snap

```diff
@@ -684,7 +684,7 @@ Object {
 
 exports[`Slack Element widgets select input user element renders with opts that override 1`] = `
 Object {
-  "action_id": "channel",
+  "action_id": "user",
```

Comment:
this is the only thing in this file that isn't just test renames - just cleanup.